### DISCUSSION TITLE
0.0.14 Beta Release

### DISCUSSION THREAD

**Discussion by Patrick762** [Patrick762/hassio-bluetti-bt#40] [Announcements]
Contains field definitions for the EP760 for testing purposes.

Please don't update if you don't want to test the EP760 with this integration.

<hr /><em>This discussion was created from the release <a href='https://github.com/Patrick762/hassio-bluetti-bt/releases/tag/0.0.14'>0.0.14 Beta Release</a>.</em>

**Comment by JADMIM** [comment]
Good afternoon,
I have replaced the previous integration with version 0.14 but it still does not work, it puts an error in the components. I don't know if it comes from the integration or from what I'm doing wrong. What I do is from studio code server I delete the entire bluetti folder, restart home and then copy the entire folder to customs again and restart home again but it gets an error.

**Reply by Patrick762** [reply]
Can you check if the Bluetooth name of your EP760 is starting with "EP760" ?
There was an update that broke the integration by renaming it to "PBOX" https://github.com/Patrick762/hassio-bluetti-bt/discussions/41

I'm already working on a fix for this, but it could take a while.

**Reply by JADMIM** [reply]
Good afternoon,

My ep760 is connected by Wi-Fi and I think it does not have a fixed IP. I will have to look at how to set the fixed IP. If there is an easier way to connect, could you tell me.

Thank you so much

**Reply by JADMIM** [reply]
and I forgot to tell you yesterday that my bluetooth name is not ep760, it starts as you say with PBOX

**Reply by Patrick762** [reply]
I'm working on a fix for that now. Next version should work for you and others with the PBOX update. The WiFi settings are not relevant because this integration is using Bluetooth only.

**Reply by Patrick762** [reply]
Next beta release is now available. Can you test if it works for your EP760? I can only test with the EP600, but it seems to work with the PBOX Bluetooth name

**Reply by JADMIM** [reply]
So to install this model, should I connect it via Wi-Fi or Bluetooth to my home assistant? If it is via Bluetooth, I must first buy an adapter to be able to connect it to my Raspberry Pi4 first.

**Reply by Patrick762** [reply]
To use the integration, you need some kind of [Bluetooth Dongle](https://www.home-assistant.io/integrations/bluetooth/#known-working-high-performance-adapters) or [ESPhome Bluetooth Proxy](https://esphome.io/components/bluetooth_proxy.html) to connect your Home Assistant to your EP760. After that the integration should search for the Powerstation automatically. I don't know if you can use the built in Bluetooth module on your Raspberry Pi.

**Reply by JADMIM** [reply]
Good morning,

I have finally configured my Bluetooth and I have already managed to integrate the ep760 into my HA, thank you very much.
I don't know if it's normal, only the battery level appears, all the other values appear unknown, that is, in the integration I can only see the battery level and not the solar production, the battery charge or the battery consumption. home
![1707910236375](https://github.com/Patrick762/hassio-bluetti-bt/assets/134491184/673921e3-bb48-48cb-8521-6ba8b6d585e6)

**Reply by Patrick762** [reply]
Can you check your Home Assistant logs? It could be possible that your station doesn't have all sensors. I'm currently working on a bigger release to also make it easier to add new devices

**Reply by JADMIM** [reply]
good morning,

Where do I look at these home assistant logs? The sensors that put unknown ones are there if I enter the bluetti application such as solar production, house consumption, grid injection,...

**Reply by JADMIM** [reply]
Este error se originó a partir de una integración personalizada.

Logger: custom_components.bluetti_bt.sensor
Source: custom_components/bluetti_bt/sensor.py:162
Integration: Bluetti BT (documentation, issues)
First occurred: 16:24:02 (1050 occurrences)
Last logged: 17:15:08

No data for available for (max_ac_output_power)
No data for available for (max_ac_output_current)
No data for available for (bcu_version)
No data for available for (safety_module_version)
No data for available for (high_voltage_module_version)

**Reply by Patrick762** [reply]
I'm looking into it. Got the same problem with my EP600 now

**Reply by Patrick762** [reply]
@JADMIM Did you see the values after some time?
Do you see the DC solar inputs?

**Reply by JADMIM** [reply]
Good morning,

I don't see anything, only the limits that I have placed on my battery and the charge it has at this moment, just like the photo I sent you.

**Comment by Patrick762** [comment]
I'll close this because the integration changed significantly in the last few versions
